envpool/sokoban/level_loader.cc

```diff
@@ -163,6 +165,7 @@ void LevelLoader::LoadFile(std::mt19937& gen) {
             << "x" << dim_room << std::endl;
         throw std::runtime_error(msg.str());
       }
+      levels_.emplace_back(std::make_pair(cur_level_idx++, cur_level));
```

Comment:
```suggestion
      levels_.push_back(std::make_pair(cur_level_idx++, cur_level));
```

will just put the object there instead of constructing a new one (originally would be copying with move-initialization, so no big deal)